### DISCUSSION TITLE
Plans for a new release?

### DISCUSSION THREAD

**Discussion by XeCycle** [aiortc/aioquic#380] [Q&A]
Hi dear maintainers, I'm affected by the problem solved in #349.  The PyPI released version is still affected; I see it is possible to workaround downstream, but do we want to release a new version with this problem fixed?

**Comment by Ousret** [comment]
Hello,

For those who stumble on that question.
Pending his return (or not), you may use the `qh3` fork that I maintain for the time being.

I am pursuing full support of HTTP 3 over QUIC in urllib3, see https://gist.github.com/Ousret/0fb70d8bb8e26d5ea494d2a51fd3920f
if you want to try it out.

I have ported numerous ideas and ported myself some.

When the original author comes back, I will propose a merge so that the fork can sunset itself. If not I will continue to maintain it as long as I can.

Links:
- https://github.com/Ousret/qh3
- https://github.com/Ousret/qh3/releases
- https://pypi.org/project/qh3/

**Comment by jlaine** [comment]
aioquic 0.9.21 is on PyPI :)
